pkg/redo/writer/writer.go

```diff
@@ -25,32 +25,31 @@ import (
 )
 
 var (
-	_ RedoEvent = (*pevent.DMLEvent)(nil)
+	_ RedoEvent = (*pevent.RedoRowEvent)(nil)
 	_ RedoEvent = (*pevent.DDLEvent)(nil)
 )
 
 // RedoEvent is the interface for redo event.
 type RedoEvent interface {
+	PostFlush()
 	ToRedoLog() *pevent.RedoLog
 }
 
 // RedoLogWriter defines the interfaces used to write redo log, all operations are thread-safe.
 type RedoLogWriter interface {
```

Comment:
could we consider we just use the sink interface, instead a new interface for redo seperately

Reply:
No. I implement `RedoRowEvent` to store every row as an event in the DML event. It is like `RowEvent`, which is used in Kafka. The `RedoRowEvent` is not necessary to implement other methods.